### PR TITLE
[MIRROR] Moths, except I have stolen them and they no longer conflict

### PR DESCRIPTION
**Pull request by ghost**
Original PR: https://github.com/tgstation/tgstation/pull/34498
--------------------
:cl: Joan and More Robust Than You
add: Adds moths! They're a new species that is able to fly in pressurized areas provided gravity is off... and that they still have wings.
/:cl:

**CREDITS TO @ tortellinitony, @ FuryMcFlurry, and @ TDcoolguy300 for their original work on moths.**

### CODE REVIEW

**Comment by Carbonhell** [comment]
Needs all those retarded advantages to be disabled before going in
Or just disable them completely and leave ours in

**Comment by Carbonhell** [comment]
With a quick glance at the code i think we can merge everything and only keep our moth.dm specie file to make everything work as desired,and better. The language could be kept,doesnt seems like a big issue.

File: code/_globalvars/lists/names.dm [hunk 1]
```diff
@@ -13,6 +13,7 @@ GLOBAL_LIST_INIT(clown_names, world.file2list("strings/names/clown.txt"))
 GLOBAL_LIST_INIT(mime_names, world.file2list("strings/names/mime.txt"))
 GLOBAL_LIST_INIT(carp_names, world.file2list("strings/names/carp.txt"))
 GLOBAL_LIST_INIT(golem_names, world.file2list("strings/names/golem.txt"))
+GLOBAL_LIST_INIT(moth_names, world.file2list("strings/names/moth.txt"))
```
Comment (GuyonBroadway) [inline code/_globalvars/lists/names.dm:16]: Reccomend we replace our local version of this file with as many variations of the work cuck as we possibly can.

**Comment by Pyko1** [comment]
merge https://github.com/HippieStation/HippieStation/pull/6089 after this

**Comment by Pyko1** [comment]
also pretty sure we don't actually need the config update.
due to hippie mothshit module overwriting the config check.

**Comment by Pyko1** [comment]
added https://github.com/HippieStation/HippieStation/pull/6116#issuecomment-359271387 fix to the code that goes in to hippie folder

**Comment by Pyko1** [comment]
added the contents of open moth fix mirrors to this PR, as merging this would had just those conflict.

**Comment by Pyko1** [comment]
fixed once again, and added changes from moth mirrors that are bound to conflict.

File: code/modules/mob/living/carbon/human/species_types/mothmen.dm [hunk 2]
```diff
@@ -0,0 +1,90 @@
+Skip to content
+This repository
+Search
+Pull requests
```
Comment (tortellinitony) [inline code/modules/mob/living/carbon/human/species_types/mothmen.dm]: fix this file

File: tgstation.dme [hunk 3]
```diff
@@ -1602,6 +1602,7 @@
 #include "code\modules\language\language_menu.dm"
 #include "code\modules\language\machine.dm"
 #include "code\modules\language\monkey.dm"
+#include "code\modules\language\moth.dm"
```
Comment (tortellinitony) [inline tgstation.dme:1605]: add files to hippiestation.dme

**Review by tortellinitony — CHANGES_REQUESTED** [review]

File: tgstation.dme (hunk 3 above)
Comment (Pyko1) [reply tgstation.dme]: actually is ok, tg removed the file on moth fix, but diditn undefine it

File: code/modules/mob/living/carbon/human/species_types/mothmen.dm (hunk 2 above)
Comment (Pyko1) [reply code/modules/mob/living/carbon/human/species_types/mothmen.dm]: wow, no idea how it even checked out with those copy paste issues,

File: code/modules/mob/living/carbon/human/species_types/mothmen.dm [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+		var/datum/gas_mixture/current = H.loc.return_air()
+		if(current && (current.return_pressure() >= ONE_ATMOSPHERE*0.85)) //as long as there's reasonable pressure and no gravity, flight is possible
+			return TRUE
+© 2018 GitHub, Inc.
+Terms
+Privacy
+Security
+Status
+Help
+Contact GitHub
```
Comment (tortellinitony) [inline code/modules/mob/living/carbon/human/species_types/mothmen.dm]: there is more

**Review by tortellinitony — DISMISSED** [review]

**Review by Carbonhell — DISMISSED** [review]

**Review by Carbonhell — APPROVED** [review]

**Comment by Carbonhell** [comment]
@Pyko1 merge it,i approve

**Pull request merged**